src/constants/questionnaireTypes.js

```diff
@@ -0,0 +1,2 @@
+module.exports.SOCIAL = "Social";
+module.exports.BUSINESS = "Business";
```

Comment:
We could consider changing this to the below to be more consistent with the rest of the codebase.

```
const SOCIAL = "Social";
const BUSINESS = "Business";

module.exports = () => {
    SOCIAL,
    BUSINESS
}
```